01-node-tutorial/answers/content/04-names.js

```diff
@@ -0,0 +1,7 @@
+// local
+const secret = 'SUPER SECRET'
+// share
+const john = 'john'
+const peter = 'peter'
```

Comment:
Challenge question: what is the difference between local and shared? How would we need to change the file to make `secret` be shared?